tsp.py

```diff
@@ -0,0 +1,43 @@
+# pip install tsp_solver
+# kindly note the code takes considerable amount of time to run depending upon the "size" parameter.
+import os  
+import cv2  
+from PIL import Image  
+import cv2
+from scipy.spatial.distance import pdist, squareform  
+from tsp_solver.greedy_numpy import solve_tsp
```

Comment:
Can you elaborate from where are you importing this? Is it a python package or you wrote a script.

Reply:
It is a Python Package, sir. Here is the [ link](https://pypi.org/project/tsp_solver/)

Reply:
The Github repo [link](https://github.com/dmishin/tsp-solver)

Reply:
Okay!